tutorials/vscodeJavaSetup.md

```diff
@@ -0,0 +1,182 @@
+{% set title="VS Code - Setting up a Java project" %}
+<frontmatter>
+  title: "{{ title }}"
+  pageNav: 2
+</frontmatter>
+
+# {{ title }}
+
+This tutorial will guide you through creating and managing Java projects in VS Code.
+
+<box type="info" seamless>
+
+This tutorial is adapted from the [official VS Code Java Tutorial](https://code.visualstudio.com/docs/java/java-tutorial).
+</box>
+
+## Install the Extension Pack for Java
+
+To develop Java applications in VS Code, you need to install the appropriate extensions that provide language support, debugging, and other essential features. The easiest way to get started with Java development in VS Code is to install the [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack), which includes all the essential extensions you need.
```

Comment:
Mention how to deal with this screen (given we recommend students to install the JDK first, they should choose `Mark Done`):
![image](https://github.com/user-attachments/assets/4e5ea827-39a7-47c7-866e-bd1bf144b138)